dev/generate

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env sh
 set -e
 
+BRANCH=${PROTO_BRANCH:-main}
+
+echo "Generating protos from branch ${BRANCH}..."
```

Comment:
What is the goal of this change? I think it's a good thing that we currently need to be very explicit about which branch the protos are generated from, e.g. if not from main then it should be clearly demarcated

Reply:
This is for development and defaults to main when not set.